src/components/modals/LogConversionConflictModal.tsx

```diff
@@ -0,0 +1,220 @@
+import { FM } from '../../react-intl-messages'
+import * as CLM from '@conversationlearner/models'
+import * as ExtractorResponseEditor from '../ExtractorResponseEditor'
+import { formatMessageId } from '../../Utils/util'
+import { injectIntl, InjectedIntlProps } from 'react-intl'
+import './LogConversionConflictModal.css'
+
+interface ReceivedProps {
+    onClose: Function
+    onAccept: (conflictPairs: ConflictPair[]) => void
+    open: boolean
+
+    title: string
+    entities: CLM.EntityBase[]
+    conflictPairs: ConflictPair[]
+}
+
+type Props = ReceivedProps & InjectedIntlProps
+
+export interface ConflictPair {
+    roundIndex: number
+    textVariationIndex: number
+    conflicting: CLM.ExtractResponse
+    previouslySubmitted: CLM.ExtractResponse
+}
+
+interface State {
+    currentConflictPairIndex: number
+}
+
+class LogConversionConflictModal extends React.Component<Props, State> {
+    state: Readonly<State> = {
+        currentConflictPairIndex: 0
+    }
+
+    componentWillReceiveProps(nextProps: Props) {
+        // Reset index when modal is opened
+        if (nextProps.open && !this.props.open) {
+            this.setState({
+                currentConflictPairIndex: 0
+            })
+        }
+    }
+
+    onClickConflictButton(index: number) {
+        this.setState({
+            currentConflictPairIndex: index
+        })
+    }
+
+    onClickPrevious = () => {
+        this.setState(prevState => {
+            const nextIndex = prevState.currentConflictPairIndex - 1
+            const minIndex = 0
+            const maxIndex = Math.max(0, this.props.conflictPairs.length - 1)
+
+            return {
+                currentConflictPairIndex: nextIndex < minIndex
+                    ? maxIndex
+                    : nextIndex
+            }
+        })
+    }
+
+    onClickNext = () => {
+        this.setState(prevState => {
+            const nextIndex = prevState.currentConflictPairIndex + 1
+            const minIndex = 0
+            const maxIndex = Math.max(0, this.props.conflictPairs.length - 1)
+
+            return {
+                currentConflictPairIndex: nextIndex > maxIndex
+                    ? minIndex
+                    : nextIndex
+            }
+        })
+    }
+
+    render() {
+        const currentConflictPair = this.props.conflictPairs[this.state.currentConflictPairIndex]
+        const { intl } = this.props
+        return (
+            <Modal
+                isOpen={this.props.open}
+                className={OF.FontClassNames.mediumPlus}
+                containerClassName="cl-modal cl-modal--medium"
+                onDismiss={() => this.props.onClose()}
+            >
+                <div
+                    className={`cl-modal_header cl-text--error ${OF.FontClassNames.xxLarge} `}
+                    data-testid="log-conversion-conflicts-modal"
+                >
+                    <OF.Icon iconName="Warning" />&nbsp;{formatMessageId(intl, FM.LOGCONVERSIONCONFLICTMODAL_TITLE)}
```

Comment:
Slight preference for using <pre> or <span> with padding over &nbsp, but not worth changing

Reply:
>   < pre > or < span >